front/package-lock.json

```diff
@@ -228,7 +228,6 @@
       "version": "2.1.2",
       "resolved": "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.1.2.tgz",
       "integrity": "sha512-hoyByceqwKirw7w3Z7gnIIZC3Wx3J484Y3L/cMpXFbr7d9ZQj2mODrirNzcJa+SM3UlpWXYvKV4RlRpFXlWgXg==",
-      "dev": true,
```

Comment:
Multiple similar changes appeared in this file after pulling master (today) and running the reset script.

I am assuming these changes should be merged to master.